BuildDatabase

```diff
@@ -325,10 +325,9 @@ $file = $seqFilename;
 if ( $engine eq "rmblast" ) {
   my $cmd = "$NCBIBLASTDB_PRGM -blastdb_version 4 -out $options{'name'} -parse_seqids -dbtype nucl -in $file 2>&1";
   print "Running: $cmd\n" if ( $DEBUG );
-  $results = `$cmd`;
-  if ( !-s "$options{'name'}.nsq" ) {
-    print
-"The makeblastdb program did not generate the\nfile $options{'name'}.nsq.  Please check your input file(s) for potential formating errors.\n$NCBIBLASTDB_PRGM returned: $results\n";
+  $results = `$cmd; echo \$?`;
+  if ( $results !~ m/0\n$/ ) {
```

Comment:
Let's move the error code checking into the perl: ``$results = `$cmd`; if ( $? ) { ... }``

Reply:
Cool, just forgot to mention that I remove the new line for that print becauses that message wasn't displaying when I first found the error, but you can test it and see if it works for you guys. Otherwise this should fix the multi-database problem :)